teachertool/src/utils/index.ts

```diff
@@ -1,13 +1,13 @@
 import { nanoid } from "nanoid";
-import { NotificationWithId } from "../types";
+import { ToastType, ToastWithId } from "../types";
 import { classList } from "react-common/components/util";
 
-export function makeNotification(message: string, duration: number): NotificationWithId {
+export function makeNotification(type: ToastType, text: string, timeoutMs: number = 5000): ToastWithId {
```

Comment:
?
```suggestion
export function makeToast(type: ToastType, text: string, timeoutMs: number = 5000): ToastWithId {
```

Reply:
For toast colors, a nice touch might be to use the rgb values from the microbit's four colors:
```css
{
    green: #3AFFB3;
    cyan: #3ADCFF;
    yellow: #FFD43A;
    red: #FF3A54;
}
```